### DISCUSSION TITLE
Help with weird message and unworking keys

### DISCUSSION THREAD

**Discussion by dachmu** [neovide/neovide#568] [Q&A] [answered]
Hi, I Have two issues I am not sure how to solve, first is this weird message on the corner of the editor that bugs me, I can't find how to get rid of it, or why is there.
![image](https://user-images.githubusercontent.com/80774220/115070648-d768b000-9eba-11eb-845f-09ab5e9884a4.png)

The next one is that most of my numpad is not working with neovide, Number keys work fine, but all other keys on the numpad simply don't do anything, this only happens inside neovide, in terminal they work just fine.

Thank you all for this great project!

**Comment by zoomlogo** [comment, marked as answer]
> The next one is that most of my numpad is not working  with neovide, Number keys work fine, but all other keys on the numpad simply don't do anything, this only happens inside neovide, in terminal they work just fine.

#445

**Reply by zoomlogo** [reply]
Don't know about the text tho.

**Reply by dachmu** [reply]
Hi, thanks!, about the text, I finally manage to stumble upon why, I had to delete C:\ProgramData\NVIDIA Corporation\Drs\nvdrssel.bin, that fixed it

**Reply by Kethku** [reply]
Woah thats wild. How did you figure out that was the issue?

**Reply by dachmu** [reply]
I ran another software, NetLogo, in 3D mode, and the message appeared there as well, so I figure out it had to be something with the graphics card, so I went to google and search it with by nvidia instead of specific software and I found the solution that way
